Index.html

```diff
@@ -0,0 +1,189 @@
+                    feedback which helps to shape the product
+                    and can quickly eliminate potential risks –
+                    saving you time and money.</p>
+
+            </article>
+            <article class="article-settings-section-3">
+                <img class="article-section-3-img" src="assets/icon_graphic-design.svg" alt="Icon Graphic Design"/>
+                <p class="title-article">Logo Design + Branding</p>
+                <p class="paragraph-article">Nike, Apple, Coca-Cola, Starbucks. What do
+                    all of these brands have in common? Easily
+                    recognizable logos and brand identity. The
+                    best brands stand out from the crowd by
+                    having a clear and unique point of view. We
+                    will work with you to understand your
+                    vision and help create a unique brand
+                    identity and style guide for your company.</p>
+
+            </article>
+            <article class="article-settings-section-3">
+                <img class="article-section-3-img" src="assets/icon_seo.svg" alt="SEO"/>
+                <p class="title-article">SEO</p>
+                <p class="paragraph-article">Do you want your site to rank higher in
+                    search results? Google is constantly
+                    changing their algorithms for how they
+                    rank pages. Don’t worry about constantly
+                    playing catch-up. We will make sure your
+                    website it optimized, providing you the best
+                    opportunities for organic search exposure.</p>
+
+            </article>
+            <article class="article-settings-section-3">
+                <img class="article-section-3-img" src="assets/icon_copywriting.svg" alt="Icon Copywriting"/>
+                <p class="title-article">Copy Writing</p>
+                <p class="paragraph-article">Brand messaging needs to be clear, direct
+                    and easy to understand to be effective. A
+                    unique tone of voice will help you stand out
+                    from the crowd and build brand loyalty. We
+                    work with you to establish your brand bible
+                    and ensure your messaging is focused and
+                    targeted.</p>
+
+            </article>
+        </div>
+
+    </section>
+
+    <section class="section-4 outer-container">
+        <div class="form-section-4 inner-container">
+            <form action="">
+                <h5>Let's work together</h5>
+                <p>Have a question? Need a quote? Let us know!</p>
+                <p>Name</p>
+                <input type="text" placeholder="Name" />
+                <p>Email</p>
+                <input type="email" placeholder="Email" />
+                <p>Message</p>
+                <input type="text" placeholder="Message" />
+                <button type="button">Send</button>
```

Comment:
Het type zou hier submit moeten zijn, anders wordt straks het formulier niet "verzonden" als we wel javaScript zouden gaan gebruiken (en dan werkt Enter ook niet!)